virtual-organizations/auger.yaml

```diff
@@ -20,13 +20,8 @@ ID: 102
 LongName: Pierre Auger Observatory
 MembershipServicesURL: https://voms1.egee.cesnet.cz:8443/voms/auger
 OASIS:
-  Managers:
-    - Name: Jiri Chudoba
-      DNs: /DC=cz/DC=cesnet-ca/O=Institute of Physics of the Academy of Sciences of
-        the CR/CN=Jiri Chudoba
-      ID: 3e40fb9b1d433ca8af3aff19a62d68d8809c3d06
   OASISRepoURLs:
-  - http://cvmfs-stratum0.gridpp.rl.ac.uk:8000/cvmfs/auger.egi.eu
+    - http://cvmfs-stratum0.gridpp.rl.ac.uk:8000/cvmfs/auger.egi.eu
   UseOASIS: true
```

Comment:
I think this one can be false too, they only have one test file.

Reply:
Done